configuration/postgres/postgres.go

```diff
@@ -69,7 +69,7 @@ const (
 	ErrorMissingConnectionString = "missing postgreSQL connection string"
 	ErrorAlreadyInitialized      = "postgreSQL configuration store already initialized"
 	ErrorMissingMaxTimeout       = "missing PostgreSQL maxTimeout setting in configuration"
-	QueryTableExists             = "SELECT EXISTS (SELECT FROM pg_tables where tablename = $1)"
+	QueryTableExists             = "SELECT EXISTS (SELECT FROM pg_tables where tablename = lower($1))"
```

Comment:
Why are you lowercasing the table?

Reply:
This is because table names in pg_tables are stored in lowercase only, and SELECT clause does the case sensitive comparison, so otherwise the query output is always false:

<img width="546" alt="image" src="https://user-images.githubusercontent.com/37222089/229603369-b4be224d-58aa-4774-ab18-2346ce336ae5.png">

Reply:
@ItalyPaleAle Please note that this would be a breaking change in postgres metadata. cc @mukundansundar

Reply:
It's an alpha component so ok to add breaking changes.